app/image.html

```diff
@@ -0,0 +1,452 @@
+        page-size="{{pageSize}}"
+        query='[[imageOffersQuery]]'
+        results="{{imageOffers}}"
+        elastic-type='["offer"]'
+        loading="{{offersLoading}}"
+        last-error="{{error}}">
+
+      </elastic-search>
+
+      <json-transform
+       data-in="[[imageOffers]]"
+       data-out="{{imageDisplay.imageOffers}}"
+       transform-function="[[clientConfig.transforms.image.imageOffersData]]"
+       ></json-transform>
+
+    <!-- Location timeline for bar chart and itinerary -->
+
+    <query-builder
+      query="{{locationTimelineQuery}}"
+      query-template="{{config.queryTemplates.locationTimeline}}"
+      field="mainEntityOfPage.hasImagePart.uri" value="[[imageDisplay.uri]]">
+    </query-builder>
+
+    <elastic-search
+      client="[[esclient]]"
+      index="[[config.elasticIndex]]"
+      query="[[locationTimelineQuery]]"
+      results="{{locationTimelineResults}}"
+      elastic-type='["offer"]'
+      loading="{{loadingLocationTimeline}}"
+      lastError="{{error}}">
+    </elastic-search>
+
+    <json-transform
+     data-in="[[locationTimelineResults]]"
+     data-out="{{imageDisplay.locationTimeline}}"
+     transform-function="[[clientConfig.transforms.seller.locationTimeline]]"
+     ></json-transform>
+
+    <!-- Main Content -->
+    <div class="entity">
+
+      <paper-header-panel class="flex" mode="waterfall-tall">
+
+        <!-- Entity header -->
+        <section class="paper-header layout horizontal">
+
+          <!-- Entity title -->
+          <div class="layout horizontal center flex">
+            <iron-icon class="iron-icon-huge entity-phone-font" icon="image:image"></iron-icon>
+            <section>
+              <div class="paper-font-display1">
+                Image
+              </div>
+            </section>
+          </div>
+          <annotate-record client="[[esclient]]"
+            annotation-index="[[config.annotationIndex]]"
+            annotation-type="[[config.annotationType]]"
+            annotation-relevant="[[config.annotationRelevant]]"
+            query-template="{{config.queryTemplates.annotationQuery}}"
+            item="{{phoneDisplay.telephone}}"
+            item-type="telephone number"
+            current-user="[[config.username]]">
+          </annotate-record>
+          <user-settings username="[[config.username]]" blur-images="{{blurImages}}"></user-settings>
+
+        </section>
+
+        <!-- Entity body -->
+        <div class="entity-page">
+
+          <div class="content">
+
+            <!-- Left Pane -->
+            <div class="flex layout vertical">
+
+              <!-- Timeline - Offers between date1 and date2 with Zoomable Barchart-->
+
+
+              <paper-material>
+                <section class="when">
+                  <iron-image style="width:400px; height:400px; background-color: lightgray;"
+                    sizing="cover" preload fade src="{{imageDisplay.url}}" blurImages class="small-blur"></iron-image>
```

Comment:
The iron-image should have its sizing property set to "contain" else images that aren't completely square will get cut-off (for example see the image http://dig.isi.edu/ht/data/0F7DC7B22098A8DBF1E259EB712EADF67A71EB34 in the dig-latest index on 52.36.12.77:8080).

Also, if you want the image to be more blurry, change the class from "small-blur" to "large-blur".